templates/etcd3.conf

```diff
@@ -15,13 +15,13 @@ data-dir: {{ etcd_data_dir }}
 wal-dir: {{ etcd_data_dir }}
 {% endif %}
 # Number of committed transactions to trigger a snapshot to disk.
-snapshot-count: 10000
+snapshot-count: {{ snapshot_count }}
```

Comment:
I like this improvement to make that `snapshot-count` managed in the charm, along with the other two